kong/observability/tracing/propagation/extractors/instana.lua

```diff
@@ -0,0 +1,67 @@
+local _EXTRACTOR        = require "kong.observability.tracing.propagation.extractors._base"
+local propagation_utils = require "kong.observability.tracing.propagation.utils"
+local from_hex          = propagation_utils.from_hex
+
+local INSTANA_EXTRACTOR = _EXTRACTOR:new({
+  headers_validate = {
+    any = {
+      "x-instana-t",
+      "x-instana-s",
+      "x-instana-l",
+    }
+  }
+})
+
+function INSTANA_EXTRACTOR:get_context(headers)
+
+  local trace_id_raw = headers["x-instana-t"]
+
+  if type(trace_id_raw) ~= "string" then
+    return
+  end
+
+  if trace_id_raw then
```

Comment:
this `if` is not needed, we know `type` is string by now, so it cannot be `nil`

Reply:
updated